### PR TITLE
SET_VEHICLE_STARTUP_REV_SOUND - N_0xf1f8157b8c3f171c

### PR DESCRIPTION
**Pull request by IllInuz**
Sets correct native translation for 
https://docs.fivem.net/natives/?_0xF1F8157B8C3F171C

### CODE REVIEW

File: AUDIO/N_0xf1f8157b8c3f171c.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
 ---
 ns: AUDIO
 ---
-## _0xF1F8157B8C3F171C
+## SET_VEHICLE_STARTUP_REV_SOUND
 
 ```c
 // 0xF1F8157B8C3F171C 0x1C0C5E4C
-void _0xF1F8157B8C3F171C(Vehicle vehicle, char* p1, char* p2);
+void SET_VEHICLE_STARTUP_REV_SOUND(Vehicle vehicle, const char* p1, const char* p2);
```
Comment (AvarianKnight) [inline AUDIO/N_0xf1f8157b8c3f171c.md:8]: Does native gen handle const char?

Comment (IllInuz) [reply AUDIO/N_0xf1f8157b8c3f171c.md:8]: is used in GetVehicleClassFromName so i figured it did, if not i'll edit.

Comment (AvarianKnight) [reply AUDIO/N_0xf1f8157b8c3f171c.md:8]: I don't see const char being used in any actual natives, only in examples or native doc ports

Comment (IllInuz) [reply AUDIO/N_0xf1f8157b8c3f171c.md:8]: Will check and verify when I got time, seems like ur right on ur note though, never payed attention to const char or char

Comment (AvarianKnight) [reply AUDIO/N_0xf1f8157b8c3f171c.md]: 👍 Also don't forget to add an alias for the old native see the [contribution guidelines](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md) for an example.

File: AUDIO/N_0xf1f8157b8c3f171c.md [hunk 2]
```diff
@@ -1,17 +1,18 @@
 ---
 ns: AUDIO
```
Comment (stannum-cfx) [inline AUDIO/N_0xf1f8157b8c3f171c.md:2]: Don't forget about the alias.

File: AUDIO/N_0xf1f8157b8c3f171c.md [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
-## _0xF1F8157B8C3F171C
+## SET_VEHICLE_STARTUP_REV_SOUND
 
 ```c
 // 0xF1F8157B8C3F171C 0x1C0C5E4C
-void _0xF1F8157B8C3F171C(Vehicle vehicle, char* p1, char* p2);
+void SET_VEHICLE_STARTUP_REV_SOUND(Vehicle vehicle, char* p1, char* p2);
 ```
 
 ```
```
Comment (stannum-cfx) [inline AUDIO/N_0xf1f8157b8c3f171c.md:11]: Please convert this into a proper description (without a code block) and add the examples (with the new native name) as described here: https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md

**Review by stannum-cfx — CHANGES_REQUESTED** [review]
Please rename the file when changing the native name.

**Pull request merged**